src/current/v22.2/performance-recipes.md

```diff
@@ -117,6 +117,8 @@ Not every full table scan is an indicator of poor performance. The [cost-based o
 
 [Examine the statements](sql-tuning-with-explain.html) that result in full table scans and consider adding [secondary indexes](schema-design-indexes.html#create-a-secondary-index).
 
+In the DB Console, visit the [**Schema Insights** tab](ui-insights-page.html#schema-insights-tab) on the [**Insights** page](ui-insights-page.html) and check if there are any insights to create missing indexes. A missing index may cause a statement to have a [suboptimal plan](ui-insights-page.html#statement-execution-details).
```

Comment:
Nit: Perhaps call out that these missing index recommendations are generated **based on slow executions**

Not all full table scans are necessarily bad as this document calls out but if the execution was slow (based on the insights threshold), then it's likely the recommendation is valid and if the plan had a full table scan, it's likely that it should be removed with an index.